epsagon/utils.py

```diff
@@ -445,3 +453,74 @@ def database_connection_type(hostname, default_type):
     if 'redshift.amazonaws' in hostname:
         return 'redshift'
     return default_type
+
+
```

Comment:
Please add tests to this function (see tests/test_utils)

Reply:
added tests